alodataset/merge_dataset.py

```diff
@@ -18,18 +20,48 @@ class MergeDataset(torch.utils.data.Dataset):
     ----------
     datasets : List[alodataset.BaseDataset]
         List of datasets
+    order : List[int]
+        How to order samples.
+            Example: MergeDataset([ds1, ds2, ds3], order=[1, 2, 1]) will order samples
+                     as follow : sample_ds1, sample_ds2, sample_ds2, sample_ds3, sample_ds1 ...
+    lim_samples : int
+        Maximum number of samples. Only when order is not None.
     transform_fn : function
         transformation applied to each sample
     weights: List[int] | None
         For N datasets, a list of N integer weights.
         The samples from a dataset with weight `w` will appear `w` times in the MergeDataset.
     """
-
-    def __init__(self, datasets, transform_fn=None, weights=None):
+    def __init__(
+            self,
+            datasets,
+            order=None,
+            shuffle=False,
+            lim_samples=None,
+            transform_fn=None,
```

Comment:
Is it still possible to use `weights` ? This is still in the doc but not use in the code anymore. Also it seem to me that weights was a good parameter to decide on how to sample from each dataset.